### DISCUSSION TITLE
Feature: Disable "Create new" Button

### DISCUSSION THREAD

**Discussion by logan-anderson** [tinacms/tinacms#3304] [Ideas]
Gauging interest on on the ability disable the create button (as seen below).

<img width="1387" alt="Screen Shot 2022-10-24 at 3 25 54 PM" src="https://user-images.githubusercontent.com/43075109/197609646-b5fe1349-9d48-4825-90a3-78bad9c2166f.png">

If a configuration option is added it would look like this.
<img width="1387" alt="Screen Shot 2022-10-24 at 3 30 27 PM" src="https://user-images.githubusercontent.com/43075109/197609821-363a9053-6fe8-482c-9dc8-067690faf802.png">

The configuration could look something like this.

```ts
collections:  [{
  name: "foo",
  label: "Foo",
  ui:{ 
       hideCreateButton: true,
  },
  fields: [
   //...
  ],
}]
```

React with a 👍 if this is a good idea, or a 👎  if you would never use it. Feel free to suggest a better API in the comments.

**Comment by jamespohalloran** [comment]
I could see disabling delete on certain collections also being handy. Maybe something like:

```
collections:  [{
  name: "foo",
  label: "Foo",
  ui:{ 
      allowedActions: {
           create: true,
           delete: false
      }
  },
  fields: [
   //...
  ],
}]
```

**Comment by coreyaus** [comment]
Love this and would definitely use it! 🙌 

There are a couple of related discussions that were created in the past that suggest it's something at least a few others are interested in too:
- https://github.com/tinacms/tinacms/discussions/2948
- https://github.com/tinacms/tinacms/discussions/2805

We already have some collections (or templates) that are only intended to be used for a single file (e.g. theme properties like `tags.json`, `colours.json`) so having a way to disable the create action on those will be an excellent addition 👌

**Comment by logan-anderson** [comment]
The ability to hide the delete and create buttons is implemented in this PR: https://github.com/tinacms/tinacms/pull/3349
